packages/nextjs/app/debug/_components/contract/ContractUI.tsx

```diff
@@ -97,7 +97,7 @@ export const ContractUI = ({
         </div>
 
         <div className="col-span-1 lg:col-span-2 flex flex-col gap-6">
-          <div className="tabs tabs-boxed">
+          <div className="tabs tabs-boxed border border-[#8A45FC]">
```

Comment:
Hey @wugalde19 , thanks alot for the PR! Really appreciate it

I personally prefer this design, but with less border radius, just to keep it consistent with the other boxes!

Reply:
by "this design" i meant option 2

Reply:
I like 2

Reply:
@Nadai2010 @metalboyrick Great
By less border radius you mean something like this: 
<img width="1028" alt="image" src="https://github.com/user-attachments/assets/5d60d4ed-4b57-4476-9c26-0ee77cfe3964">

Note: I'm using the same 5px (for both the tabs container and the active tab) that the other boxes use

Reply:
I just happened to upload this change, xd very good, you can check the tone of the background a little, in the light you see a light gray/blue tone, and in the dark a dark gray, try to make it follow the line with the rest, (maybe transparent)

Reply:
Transparent sounds like a good idea
Just push the latest changes @Nadai2010 
Wanna take a look?

Reply:
I'm checking now